File: jdroid-android-core/src/main/java/com/jdroid/android/linkedin/LinkedInHelper.java
package com.jdroid.android.linkedin;

import android.content.Intent;
import android.net.Uri;

import com.jdroid.android.activity.ActivityLauncher;

public class LinkedInHelper {
	
	public static void openCompanyPage(String companyPageId) {
		ActivityLauncher.startActivityNewTask(new Intent(Intent.ACTION_VIEW, Uri.parse("https://www.linkedin.com/company/" + companyPageId)));
	}
}

File: jdroid-android-core/src/main/java/com/jdroid/android/linkedin/LinkedInHelper.kt
package com.jdroid.android.linkedin

import android.content.Intent
import android.net.Uri

import com.jdroid.android.activity.ActivityLauncher

object LinkedInHelper {

    fun openCompanyPage(companyPageId: String) {
        ActivityLauncher.startActivityNewTask(Intent(Intent.ACTION_VIEW, Uri.parse("https://www.linkedin.com/company/$companyPageId"))
        )
    }
}
